test/worker_specific_task_queues/file_processing_workflow_test.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+require 'test'
+require 'worker_specific_task_queues/file_processing_workflow'
+require 'worker_specific_task_queues/normal_activities'
+require 'securerandom'
+require 'temporalio/testing'
+require 'temporalio/worker'
+
+module WorkerSpecificTaskQueues
+  module WorkerSpecificActivities
```

Comment:
Would consider putting these mock activities as children of the `FileProcessingWorkflowTest` class and not in a module